util/dvsim/FlowCfg.py

```diff
@@ -392,6 +394,19 @@ def print_list(self):
         for item in self.cfgs:
             item._print_list()
 
+    # function to prune only selected cfgs to build and run
+    # it will return if the object is not a master_cfg or -select_cfgs is empty
+    def prune_selected_cfgs(self):
+        if self.is_master_cfg == 0 or self.select_cfgs == []:
```

Comment:
Perhaps `not` can be used here instead of `== 0 / ==[]`.

Reply:
Thanks Sri, yup, `not` works here. Updated as you recommend.